lib/db.js

```diff
@@ -152,14 +154,14 @@ function Db(databaseName, topology, options) {
     options: options,
     // Logger instance
     logger: Logger('Db', options),
-    // Get the bson parser
-    bson: topology ? topology.bson : null,
     // Unpack read preference
     readPreference: ReadPreference.fromOptions(options),
     // Set buffermaxEntries
     bufferMaxEntries: typeof options.bufferMaxEntries === 'number' ? options.bufferMaxEntries : -1,
-    // Set up the primary key factory or fallback to ObjectID
-    pkFactory: options.pkFactory || ObjectID,
+    // Parent db (if chained)
+    parentDb: options.parentDb || null,
+    // Set up the primary key factory or fallback to ObjectId
+    pkFactory: options.pkFactory || ObjectId,
     // Get native parser
     nativeParser: options.nativeParser || options.native_parser,
```

Comment:
It occurs to me that we should also remove the notion of a `nativeParser`, it's not even used outside of a single test.